src/pages/ExamsPage/messages.js

```diff
@@ -290,6 +290,26 @@ const messages = defineMessages({
     defaultMessage: 'Please select an exam from the dropdown above.',
     description: 'A prompt to select an exam before being able to open the external review dashboard.',
   },
+  allowanceDashboardTabTitle: {
+    id: 'ExamsPage.allowanceDashboardTabTitle',
+    defaultMessage: 'Allowances',
+    description: 'Title for the allowances tab',
+  },
+  allowanceButton: {
+    id: 'ExamsPage.allowanceButton',
+    defaultMessage: 'Add allowance',
+    description: 'Label for for a button to add an allowance',
```

Comment:
```suggestion
    description: 'Label for a button to add an allowance',
```
Nit: just noticed there's an extra for in the description.

Reply:
Thank you!